pkg/virt-config/feature-gates.go

```diff
@@ -78,6 +78,9 @@ const (
 	CommonInstancetypesDeploymentGate = "CommonInstancetypesDeploymentGate"
 	// AlignCPUsGate allows emulator thread to assign two extra CPUs if needed to complete even parity.
 	AlignCPUsGate = "AlignCPUs"
+
+	// VolumesUpdateStrategy enables to specify the strategy on the volume updates.
+	VolumesUpdateStrategy = "VolumesUpdateStrategy"
```

Comment:
It depends what we want to feature gate. This could cover all the volume update strategies. Like the hotplug and migration could go behind this feature gate, or we need to introduce a feature gate for each features. @acardace wdyt?

Reply:
I'm in favor of adding a separate FG. I don't think covering this behind the vmRolloutStrategy FG is the best option, while they are correlated (volume updates depend on rollout strategy) they solve different problems.

What if we see this API is not optimal and want to change it and the rolloutStrategy graduates to GA in the meanwhile? I think it's safer to add another FG, anyway they're cheap and easy to remove if we see that things work well.

Reply:
I think the API field is fine on its own and there can be featuregates for the individual features (migration, declarative volume hotplug, etc)

Reply:
@mhenriks I think here it is more for the updateVolumeStrategy field for the VMI that could be potentially dropped

Reply:
Oh I see sure, so when storage migration is implemented there will be another feature gate for that as well?

Reply:
After this discussion, I guess so